### DISCUSSION TITLE
Minio - Files inside a Bucket can inherit his permissions?

### DISCUSSION THREAD

**Discussion by dmendozal** [minio/minio#15621] [Q&A] [answered]
Is it possible that Minio creates files with the same permissions as the bucket?
I mean, this is my bucket and his permissions.
![bucket](https://user-images.githubusercontent.com/32149661/187529545-79a7ce90-a613-4787-81cd-965bbe344e73.png)

I would like that my new files created by minio, it has the same permission as the bucket.
Currently, files are being created with the parent bucket's permission.
Here an example
![selfie](https://user-images.githubusercontent.com/32149661/187536124-6a5a4f6f-5b3c-4673-a79f-517deaa503da.png)

**Comment by harshavardhana** [comment, marked as answer]
FS permissions are not our concern. We cannot honor such behavior and it is out of our scope.
